plugin/output/file/files.go

```diff
@@ -0,0 +1,44 @@
+package file
+
+import (
+	"github.com/ozonru/file.d/logger"
+	"github.com/ozonru/file.d/pipeline"
+)
+
+// Plugins is an abstraction upon multiple file.Plugin, which helps reuse it.
+type Plugins struct {
+	plugins map[string]*Plugin
+}
+
+func NewFilePlugins(plugins map[string]*Plugin) *Plugins {
+	return &Plugins{plugins: plugins}
+}
+
+func (p *Plugins) Out(event *pipeline.Event, selector pipeline.PlugSelector) {
+	switch selector.CondType {
+	case pipeline.ByNameSelector:
+		p.plugins[selector.CondValue].Out(event)
+	case pipeline.UnknownSelector:
+		logger.Fatalf("PlugSelector type didn't set for event: %v, selector: %v", event, selector)
+	}
+}
+
+// Start runs all plugins.
+func (p *Plugins) Start(starterData pipeline.PluginsStarterMap) {
+	for plugName, plug := range p.plugins {
+		plug.Start(starterData[plugName].Config, starterData[plugName].Params)
+	}
+}
+
+// Stop stops all plugins (very useful comment).
+func (p *Plugins) Stop() {
+	for _, plug := range p.plugins {
+		plug.Stop()
+	}
+}
+
+// Exists asks if such file.Plugin exists in Plugins.
+func (p *Plugins) Exists(plugName string) (ok bool) {
```

Comment:
maybe add tests on it ?

Reply:
I'll write couple, logic here pretty simple.

Reply:
Added